src/event-listeners/onPageComplete.js

```diff
@@ -5,11 +5,19 @@ export default store => pageIndex => {
   if (state.viewer.isAccessibleMode) {
     core.getDocument().loadPageText(pageIndex, text => {
       const textContainer = document.createElement('div');
+      const className = 'accessible-text';
```

Comment:
Hmm this container has an id `pageText${pageIndex}`, so can we use `getElementById` to select it?

Reply:
yes. thanks for pointing that out. i have made the fix.